sorc/link_workflow.sh

```diff
@@ -10,7 +10,7 @@ function usage() {
 Builds all of the global-workflow components by calling the individual build
   scripts in sequence.
 
-Usage: ${BASH_SOURCE[0]} [-h][-o][--nest]
+Usage: ${BASH_SOURCE[0]} [-h][-o]
```

Comment:
@guoqing-noaa 
I don't think linking 2 sets of fix files is a good idea.
We should target consolidation rather than duplication.

Reply:
Thanks for the comment, @aerorahul !  I am open to a suitable solution.

Our experiences showed that it is troublesome that we need to relink fix files when one wants to transition from running nesting experiments to running a regular non-nest experiment. Most times we forgot to do that and got crashes. And we cannot relink those fix files in the process of a launched experiment and have to wait for its completion.
Also, this means we cannot run nesting experiments and non-nesting experiments at the same time.

My current solution is to link all directories in fix to fix/nest but update fix/nest/orog and fix/nest/ugwd. 
To my understanding,  this does not cause any harm to the current global workflow. Let me know if you find any problems with this. Thanks!

Reply:
@aerorahul Do you mean we keep the "--nest" option so that the solution only applies to those who would run nesting experiments?